go.mod

```diff
@@ -13,6 +14,7 @@ require (
 	github.com/spf13/viper v1.12.0
 	golang.org/x/oauth2 v0.0.0-20220608161450-d0670ef3b1eb
 	golang.org/x/sync v0.0.0-20220601150217-0de741cfad7f
+	golang.org/x/xerrors v0.0.0-20220517211312-f3a8303e98df
```

Comment:
Note to self: I don't think we need this, we can use stdlib `errors` because we support go1.13 <